spec/features/smart_answer/find_coronavirus_support_flow_spec.rb

```diff
@@ -0,0 +1,249 @@
+require "rails_helper"
+
+RSpec.feature "FindCoronavirusSupportFlow", type: :feature do
+  def start_flow_to_need_help_with
+    visit "/find-coronavirus-support"
+    expect(page).to have_selector("h1", text: headings[:flow_title])
+
+    click_govuk_start_button
+    expect(page).to have_selector("h1", text: headings[:need_help_with])
+  end
+
+  def continue_to_results
+    click_button "Continue"
+    expect(page).to have_selector("h1", text: headings[:nation])
+
+    choose "England"
+    click_button "Continue"
+    expect(page).to have_selector("h1", text: headings[:results])
+  end
+
+  let(:headings) do
+    {
+      flow_title: "Find out what support you can get if you’re affected by coronavirus",
+      need_help_with: "What do you need help with because of coronavirus?",
+      feel_unsafe: "Do you feel unsafe where you live?",
+      afford_rent_mortgage_bills: "Are you finding it hard to pay your rent, mortgage or bills?",
+      afford_food: "Are you finding it hard to afford food?",
+      get_food: "Are you able to get food?",
+      self_employed: "Are you self-employed, a freelancer, or a sole trader?",
+      have_you_been_made_unemployed: "Have you been made redundant or told to stop working?",
+      worried_about_work: "Are you worried about going in to work?",
+      are_you_off_work_ill: "Are you off work because you’re ill or self-isolating?",
+      have_somewhere_to_live: "Do you have somewhere to live?",
+      have_you_been_evicted: "Have you been evicted?",
+      mental_health_worries: "Are you worried about your mental health, or another adult or child’s mental health?",
+      nation: "Where do you want to find information about?",
+      results: "Information based on your answers",
+    }
+  end
+
+  let(:result_subheadings) do
+    {
+      feeling_unsafe: "Feeling unsafe",
+      paying_bills: "Paying your rent, mortgage, or bills",
+      getting_food: "Getting food",
+      being_unemployed: "Being made redundant or unemployed, or not having any work",
+      going_to_work: "Being worried about working, or off work because you’re self-isolating",
+      somewhere_to_live: "Having somewhere to live",
+      mental_health: "Mental health and wellbeing",
+    }
+  end
+
+  let(:need_help_with) do
+    {
+      feeling_unsafe: "Feeling unsafe where you live, or what to do if you’re worried about the safety of another adult or child",
+      paying_bills: "Paying your rent, mortgage, or bills",
+      getting_food: "Getting food",
+      being_unemployed: "Being made redundant or unemployed, or not having any work",
+      going_to_work: "Being worried about working, or off work because you’re self-isolating",
+      somewhere_to_live: "Having somewhere to live",
+      mental_health: "Mental health and wellbeing, including information for children",
+      none: "None of these",
+    }
+  end
```

Comment:
I understand why you've had to do this, but having to repeat these here feels wrong.  Should we not be making use of the `locales` files instead?